### ISSUE TITLE
TypeError: listen_for_events() got an unexpected keyword argument 'url'

### ISSUE THREAD

**Issue by retpoline** [CaliDog/certstream-python#39]
```
pip install certstream

$ python examples/echo.py 
Traceback (most recent call last):
  File "echo.py", line 25, in <module>
    certstream.listen_for_events(print_callback, url='wss://certstream.calidog.io/')
TypeError: listen_for_events() got an unexpected keyword argument 'url'

```
Is there something else missing to make the echo example work?

Thanks!

**Comment by Fitblip** [comment]
Howdy, I suspect this is the `websocket` library having changed. It looks like I didn't actually pin it to a specific version though https://github.com/CaliDog/certstream-python/blob/master/requirements.txt#L1 so that'd cause the breakage. I'll take a look when I have a chance, sorry for the breakage!

**Comment by Fitblip** [comment]
This should be fixed with #42
